components/catalogs/back/src/ServiceAPI/Model/Catalog.php

```diff
@@ -10,11 +10,21 @@
  */
 final class Catalog
 {
+    public const ENABLED = 'enabled';
+    public const DISABLED = 'disabled';
+    public const STATES = [self::DISABLED, self::ENABLED];
```

Comment:
```suggestion
    public const ENABLED = 'enabled';
    public const DISABLED = 'disabled';
    public const STATES = [self::DISABLED, self::ENABLED];
```

IHMO, should be private, it shouldn't leak outside your model.